pkg/controller/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -345,11 +345,19 @@ func (r *ReconcileToolchainStatus) membersHandleStatus(logger logr.Logger, toolc
 		members[memberCluster.Name] = memberStatus
 	}
 
-	// add member cluster statuses to toolchainstatus
-	ready = compareAndAssignMemberStatuses(logger, toolchainStatus, members) && ready
+	// add member cluster statuses to toolchainstatus, and assign apiEndpoint in members
+	ready = compareAndAssignMemberStatuses(logger, toolchainStatus, members,memberClusters) && ready
```

Comment:
What IDE are you using? Configure it to use `go fmt` to automatically format the code :)

Reply:
i am using goLand, will do